src/main/resources/feedbackQuestionConstSumSubmissionFormTemplate.html

```diff
@@ -1,14 +1,6 @@
-<div class="col-sm-6 width-auto">
+<div id="constSumSubmissionFormOptionFragment-${questionIndex}" class="col-sm-6 width-auto">
```

Comment:
I think this issue is due to the class settings of contrib question instead. This problem exists in the master branch as well. The reason it is apparent now is because I am displaying the text boxes on a new line. The problem occurs due to unnecessary padding being added to these text boxes. I think the intention was to remove that by adding `mobile-no-padding` as shown below but it is ineffective:

![image](https://user-images.githubusercontent.com/11634699/40841537-b6083448-65a2-11e8-9ac6-a048311cfbd8.png)

I can fix this rn but I was wondering if this should be part of another issue ?

Reply:
I can add another css rule to override the default padding of columns ?

Reply:
there are many ways to tackle this problem i suppose. What we want is just the indentation for the label and input box to be consistent across different question types.